packages/react/src/components/Masthead/Masthead.js

```diff
@@ -245,6 +245,7 @@ const Masthead = ({
                 title={title}
                 eyebrowText={eyebrowText}
                 eyebrowLink={eyebrowLink}
+                navigation={mastheadData}
```

Comment:
These are all L1 data. I'm wondering if we should consolidate this prop that goes into `Masthead` as simply `mastheadL1`, and in that the props are:

```
{
   title: 'L1 Title',
   eyebrowText: 'Eyebrow Text'
   eyebrowLink: 'https://www.ibm.com/something',
  navigation: { ... nav data ... }
}
```

Thoughts @kennylam @guilhermelMoraes ?

Reply:
@jeffchew @kennylam  thinking about it once again, I wonder if `navigation L1` is right. There are more validations to come if we choose that way. I think a better - and easier - approach would be something like: 
`<Masthead enableL1 />`. Since the Masthead will have only navigation data in one place (whether `TopNav` or `L1`), there is no need to distinguish it between `navigation` and `navigationL1`. As long as we define in the docs that this will happens, I believe this is a better approach.

Reply:
We wouldn't want the L0 navigation data in the L1 space though, if the custom data isn't set there is a chance thatthat could happen. Also, we might want to leave room in case the L1 data structure ends up being different from L0, and also if design comes back and decides to have L0 and L1 nav data co-exist.